collections/guides/use/reveal-card-data-with-elements.md

```diff
@@ -0,0 +1,169 @@
+
+The following steps address how you can create an `expiring` application:
+
+### Enable "Create Expiring Application" on a Private Application
+
+Applications of the `expiring` type are created from `private` applications, instead of a `management` application like other application types. This is done so the `expiring` application permissions are initially inherited from the parent `private` application, but still with the possibility of scoping permissions further down during creation. 
+The `private` application in question must either have the `token:read` <a href="https://developers.basistheory.com/concepts/access-controls/#permissions">permission</a> or an [access rule](https://developers.basistheory.com/concepts/access-controls/#what-are-access-rules) that has the permission in order for revealing token values to be possible. 
+To enable a `private` application to create expiring applications, simply set the toggle in the Customer Portal, either during the `private` application creation or when updating an existing one: 
+
+![Image of Create Expiring Application toggle in the portal](/assets/images/elements_reveal/enable_expiring_applications.png)
+
+Additionally, you can also enable this through our [API or SDKs](https://docs.basistheory.com/#applications-create-application) using the `can_create_expiring_applications` flag. 
+
+<span class="base-alert warning">
+  <span>
+    Tokens of type <code>card</code> are automatically added to the <code>/pci/high/</code> container.
+    Your private application created/updated needs <code>read</code> access to this container to continue with this tutorial.
+    If you don't have a private application yet that meets this criteria yet, you can <a href="https://portal.basistheory.com/applications/create?application_template_id=e6d4c554-6703-4bbb-b351-42cd2ee5cb5a">click here</a> to create one.
+  </span>
+</span>
+
+### Create The Expiring Application
+
+Now that the `private` application has the ability to create `expiring` applications, you can use the `private` application API key to create expiring applications from your server-side code.
+
+The example below uses the Basis Theory JS SDK to accomplish this:
+
+```jsx
+import { BasisTheory } from '@basis-theory/basis-theory-js';
+
+const bt = await new BasisTheory().init(process.env.BASIS_THEORY_PRIVATE_KEY);
+
+const expiringApplication = await bt.applications.create({
+  type: 'expiring',
+  expires_at: ttl(),
+  rules: [
+    {
+      description: 'Reveal Card',
+      priority: 1,
+      transform: 'reveal', // required for reveal
+      container: '/pci/high/',
+      conditions: [
+        {
+          attribute: 'id',
+          operator: 'EQUALS',
+          value: req.body?.tokenId, // from the request made to the server-side code
```

Comment:
This example is out of the demo app and it uses the NextJS API. 
They can see the rest of the code in the repository, I don't like adding it explicitly to the guide because its full of assumptions on a user's infrastructure. This is an advanced feature and I feel like how to build an API is not something we need to have in a guide.

As far as the frontend code snippet, I have it if you look at the final example code in the last section:

```js // fetching the expiring application key from your server side code
    const {
      data: { expiringKey },
    } = await axios.post('/api/authorize');```